client/gutenberg/extensions/presets/jetpack/utils/get-extensions.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import extensionSlugsJson from '../index.json';
+import { isEnabled } from 'config';
```

Comment:
This currently breaks wp-admin, where we're not exposing `window.configData`.

Reply:
Asked the following in Slack (p1545309825038100-slack-calypso-framework):

> Question: I'd like to rely on a feature flag from within a Jetpack Gutenberg block. This works within the context of Calypso, but breaks if the block is built for wp-admin, since we're not exposing `window.configData`. TBH I'd forgotton that we're relying on a `window.` global. I was wondering if there was any point to that? I've personally never live-tweaked feature flags by manipulating that global in the console. For the time being, I'm considering adding that `configData` (at least parts) to `window` in `wp-admin`. For the long-term solution: What about evaluating `config.isEnabled()` checks during build time (i.e. using a Babel plugin) instead?